newEvaluation.html

```diff
@@ -69,7 +70,7 @@
 	<!-- Section5 -->
 	<section>
 		<h5>Workflow, Leadership & Soft Skills</h5>
```

Comment:
Weird... why might this `&` character be showing up on red background in GitHub? :P